src/Helper.js

```diff
@@ -119,6 +132,7 @@ class Helper {
    */
   static async initialize(info) {
     this.#hassStates = info.hass.states;
+    this.#narrow = info.narrow;
```

Comment:
Are you sure the info object has a narrow property?
```
Object { config: {…}, hass: {…} }
​
config: Object { strategy: {…}, views: [] }
​hass: Object { auth: {…}, connection: {…}, connected: true, … }
​```

Reply:
Sure, I have tested it. As you see in the screenshots.
There are also a lot of occurrences of it in the ha frontend code: https://github.com/search?q=repo%3Ahome-assistant%2Ffrontend%20narrow&type=code

Reply:
Yeah, I've seen it also at different places, but still... it's `undefined` at my HASS.

Reply:
This might be a problem: https://github.com/home-assistant/frontend/pull/18201

Reply:
Ouch. Very strange decision from them IMO. Since they are still using narrow internally.
Quick browse of their code shows that this could work:
```
this.#narrow = window.matchMedia("(max-width: 870px)").matches;
```
ref: https://github.com/home-assistant/frontend/blob/589e3b63c71c6434f2375a3614827b2feedba8e8/src/layouts/home-assistant-main.ts#L53

Reply:
I think that's the way to go.
You should research which of the media-queries will fit best for the "narrow" mode.
max 870px seems quite large to me.

Reply:
870px suits what I have tested, and is also the definition of narrow in home assistant. I'll go with that, unless you tell me to go with something else 😄

Reply:
Updated now

Reply:
Since `narrow` is out, I'd say not to check for `undefined` and just define it ourselves the way you suggested.

Reply:
fixed